blog-homepage/src/App.js

```diff
@@ -0,0 +1,30 @@
+import './week2.css';
+import yourArticles from './_data/your-articles.json';
+import missedArticles from './_data/missed-articles.json';
+import Section from './week2app/Section';
+import Article from './week2app/Article';
+
+function App()
+{
+  const forYouPrefix = 'fy'
+  const missedItPrefix = 'mi'
```

Comment:
Nice job using a prefix in order to have different styling for the different article sections 👍